apps/guardian-ui/src/admin/FederationAdmin.tsx

```diff
@@ -98,7 +109,11 @@ export const FederationAdmin: React.FC = () => {
               {t('federation-dashboard.placeholder-fed-description')}
             </Text>
             <Flex gap='12px' mt='13px'>
-              <Pill text='Guardians' status={`${guardians}`} />
+              <Pill
+                text='Guardians'
+                status={`${guardians}`}
+                color={statusColor}
+              />
               <Pill text='Server' status='Healthy' />
               <Pill text='Uptime' status='100%' />
```

Comment:
Do these need to have `color="green"` added?

Reply:
Good call. They're static values so I'd love to remove them but 🤷‍♂️. 
https://github.com/fedimint/ui/issues/148 for tracking. Need to please bring this up in the dev call.

CC: @justinmoon in case I can't make the call.